themes/reactos/layouts/partials/navbar.html

```diff
@@ -8,10 +8,7 @@
 		<div class="col-md-offset-1 col-md-10">
 			<div class="navbar-header">
 				<a class="navbar-brand home" href="{{ .Site.BaseURL }}">
-					<img src="{{ .Site.BaseURL }}{{ .Site.Params.logo }}" alt="{{ .Title }} logo"
-						class="hidden-xs hidden-sm">
-					<img src="{{ .Site.BaseURL }}{{ .Site.Params.logo_small }}" alt="{{ .Title }} logo"
-						class="visible-xs visible-sm">
+					<img src="{{ .Site.BaseURL }}{{ .Site.Params.logo }}" alt="{{ .Title }} logo" style="height: 54px; width: 227px;">
```

Comment:
Problem basically is that we're enforcing Site.Params.logo por any kind of resolution. Reactos_0.png seems to be a 227x54 image, and this is cool for small devices but not for bigger resolutions.
The trick here is to use "srcset" instead "src": <img srcset=" {{ .Site.BaseURL }}{{.Site.Params.logo}} 1x, {{.site.BaseUrl }}{{.Site.Params.biggerlogo}} 2x" src="{{.site.BaseUrl }}{{.Site.Params.biggerlogo}}" alt="{{ .Title }}>
And a Bigger Logo is needed. This is the easiest trick, otherwise we would need to use "size".

Reply:
We probably need to use SVG then.